### PR TITLE
NewmanHangman initialization

### PR DESCRIPTION
**Pull request by Urban158**
https://github.com/loyno-cosc-a211-201901/cosc-a211-term-project-Urban158/compare?expand=1

### CODE REVIEW

File: NewmanHangman.cpp [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+#include <iostream>
+using namespace std;
+
+int main()
+{
+/*
+Given the user is expected to spell a word before enough wrong guesses draws a stick figure and gallows,
+I am not sure how the word to be spelled will be presented to the user.  Should the user open another file
+containing a list of words of which one will be randomly chosen.
```
Comment (ADWolfe-LoyNO) [inline NewmanHangman.cpp:18]: The "file of words" approach is good

File: NewmanHangman.cpp [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+Next thing I must learn to do is learn to draw to the stick figure and gallow to be used in the program.
+Once I learn how to draw the stick figure, I will have to convert to a loop file where each wrong guess will
```
Comment (ADWolfe-LoyNO) [inline NewmanHangman.cpp:21]: don't do anything fancy - it's something you can do simply

File: NewmanHangman.cpp [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/*Program Name: Hangman
+Purpose: I am hoping to write a c++ program to imitate the classic game called Hangman.
+I have been advised that the user will have several different words which may be used for
+the game.  I assumme I will eventually learn to draw using c++ and develop a funtion that will
+draw a stick figure gallow with a stick figure person attached to. The user is expected to
```
Comment (ADWolfe-LoyNO) [inline NewmanHangman.cpp:5]: you definitely want to do this as a "draw-gallows" function

File: NewmanHangman.cpp [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+Given the user is expected to spell a word before enough wrong guesses draws a stick figure and gallows,
+I am not sure how the word to be spelled will be presented to the user.  Should the user open another file
+containing a list of words of which one will be randomly chosen.
+
+Next thing I must learn to do is learn to draw to the stick figure and gallow to be used in the program.
+Once I learn how to draw the stick figure, I will have to convert to a loop file where each wrong guess will
+draw a portion of the "hangman".
+
+Should I interact with the user after each guess either with a "congratulation, you guessed a correct alphabet" or
+"sorry, guess again" and drraw a portion of the "hangman".
```
Comment (ADWolfe-LoyNO) [inline NewmanHangman.cpp:25]: yes, that is good

**Review by ADWolfe-LoyNO — APPROVED** [review]
This is a good start.  Please proceed!

**Comment by Urban158** [comment]
Got it and thanks!
Derris!

**Pull request merged**